### PR TITLE
MAUI-5096 : Prepare Overview and Getting Started UG for .NET MAUI Popup

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared Overview UG for .NET MAUI Popup

### CODE REVIEW

**Review by Lakshminatarajan — APPROVED** [review]
Approved

File: MAUI/Popup/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+                .UseMauiApp<App>()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                });
+
+            builder.ConfigureSyncfusionCore();
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:75]: Highlight the code

File: MAUI/Popup/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight c# tabtitle="MauiProgram.cs" %}
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Microsoft.Maui.Hosting;
+using Syncfusion.Maui.Core.Hosting;
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:59]: Highlight the namespace

File: MAUI/Popup/getting-started.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:75]: resolved

File: MAUI/Popup/getting-started.md (hunk 2 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:59]: resolved

File: MAUI/Popup/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+namespace GettingStarted
+{
+    public partial class MainPage : ContentPage
+    {
+        SfPopup popup;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            popup = new SfPopup();
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:121]: Highlight code

File: MAUI/Popup/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+        }
+
+        private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+        {
+            popup.Show();
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:126]: Highlight code

File: MAUI/Popup/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+* `Absolute Positioning`: Use the `SfPopup.Show(x-position, y-position)` to display the SfPopup at the specified X and y position.
+* `Relative Positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition)` to display the SfPopup at any of the 8 positions relative to the specified view.
+* `Absolute relative positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition,x position,y position)` to display the SfPopup at an absolute x,y coordinate from the relative position of the specified view.
+
+## Customizing layouts
+
+By default, you can choose a layout from the following available layouts in the SfPopup by using the property `SfPopup.AppearanceMode`.
+
+* `OneButton`: Shows the SfPopup with one button in the footer view. This is the default value.
+* `TwoButton`: Shows the SfPopup with two buttons in the footer view.
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:157]: Add code snippet for setting AppearanceMode (Two Button)  and Screenshot.

File: MAUI/Popup/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+             Padding="0,40,0,0"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup">
+    <StackLayout>
+        <Button x:Name="clickToShowPopup"
+                Text="ClickToShowPopup"
+                VerticalOptions="Start"
+                HorizontalOptions="FillAndExpand"
+                Clicked="ClickToShowPopup_Clicked" />
+        <sfPopup:SfPopup x:Name="popup">
+            <sfPopup:SfPopup.ContentTemplate>
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:183]: Highlight here

File: MAUI/Popup/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+            {
+                popupContent = new Label();
+                popupContent.Text = "This is the Customized view for SfPopup";
+                popupContent.BackgroundColor = Color.LightSkyBlue;
+                popupContent.HorizontalTextAlignment = TextAlignment.Center;
+                return popupContent;
+            });
+
+            // Adding ContentTemplate of the SfPopup
+            popup.ContentTemplate = templateView;
```
Comment (PiruthivirajM) [inline MAUI/Popup/getting-started.md:220]: Highlight this line alone

**Review by PiruthivirajM — COMMENTED** [review]
Address the review comment

File: MAUI/Popup/getting-started.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:126]: Highlighted the code

File: MAUI/Popup/getting-started.md (hunk 5 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:157]: added screenshots and code snippet

File: MAUI/Popup/getting-started.md (hunk 7 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:273]: resolved

File: MAUI/Popup/overview.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+# MAUI Popup (SfPopup) Overview
+
+Popup layout provides a layout that allows users to display an alert message with customizable buttons or load any desired view inside a popup. It provides options to fully customize the popup with custom header, body, and footer capabilities, allowing you to display important information to use precisely how user want to.
+
+## Key features
+
+* Load complex views and layouts as content.
+* Display popups with minimalistic code by calling only a single method.
+* Choose from default layouts, such as one-button layout and two-button layout, to display simple and minimalistic information.
```
Comment (Chrispineagunja) [inline MAUI/Popup/overview.md:18]: * Choose from default layouts, such as one-button and two-button layouts, to display simple and minimalistic information.

File: MAUI/Popup/overview.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+title: About .NET MAUI Popup control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+--- 
+
+# MAUI Popup (SfPopup) Overview
+
+Popup layout provides a layout that allows users to display an alert message with customizable buttons or load any desired view inside a popup. It provides options to fully customize the popup with custom header, body, and footer capabilities, allowing you to display important information to use precisely how user want to.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/overview.md:12]: modified

File: MAUI/Popup/overview.md (hunk 8 above)
Comment (RiyashameedM) [reply MAUI/Popup/overview.md:18]: modified

File: MAUI/Popup/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+title: Getting Started with .NET MAUI Popup control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Getting Started with .NET Maui Popup (SfPopup)
+
+This section provides a quick overview of how to get started with the .NET Maui Popup (SfPopup) for Maui. Walk-through the entire process of creating the real world SfPopup.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+            return builder.Build();
+        }
+    }
+}
+{% endhighlight %} 
+{% endtabs %}
+
+## Displaying popup
+
+You can display popup over your view by calling the `SfPopup.Show` method.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Popup with default appearance](Images/getting-started//maui-popup-with-default-appearance.png)
+
+You can download the source code of this sample [here](https://github.com/SyncfusionExamples/getting-started-.net-maui-popup).
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,287 @@
...
+The following list of options are available to position the SfPopup in the desired position:
+
+* `Center Positioning`: Use the `SfPopup.IsOpen` property or `SfPopup.Show` method to display the SfPopup at the center.
+* `Absolute Positioning`: Use the `SfPopup.Show(x-position, y-position)` to display the SfPopup at the specified X and y position.
+* `Relative Positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition)` to display the SfPopup at any of the 8 positions relative to the specified view.
+* `Absolute relative positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition,x position,y position)` to display the SfPopup at an absolute x,y coordinate from the relative position of the specified view.
+
+## Customizing layouts
+
+By default, you can choose a layout from the following available layouts in the SfPopup by using the property `SfPopup.AppearanceMode`.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,287 @@
...
+
+* `OneButton`: Shows the SfPopup with one button in the footer view. This is the default value.
+* `TwoButton`: Shows the SfPopup with two buttons in the footer view.
+
+You can also customize the entire view of the popup by loading the templates or custom views for the header, body, and footer.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,287 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Popup with Appearance Mode](Images/getting-started//maui-popup-with-appearance-mode.png)
+
+###  Load your template view in the popup body
+
+Any view can be added as popup content by using the `SfPopup.ContentTemplate` property to refresh it. Refer to the following code example in which a label is added as a popup content.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,287 @@
...
+
+You can download the source code of this sample [here](https://github.com/SyncfusionExamples/getting-started-.net-maui-popup).
+
+N> Two Popups cannot be displayed at the same time in a page.
+
+## Customize positioning
+
+The .NET MAUI Popup (SfPopup) allows showing the popup content at various positions.
+
+The following list of options are available to position the SfPopup in the desired position:
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,287 @@
...
+
+# Getting Started with .NET Maui Popup (SfPopup)
+
+This section provides a quick overview of how to get started with the .NET Maui Popup (SfPopup) for Maui. Walk-through the entire process of creating the real world SfPopup.
+
+
+## Creating an application using the .NET MAUI Popup
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPopup to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Popup and then install it.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md]: modified

File: MAUI/Popup/getting-started.md [hunk 18]
```diff
@@ -0,0 +1,287 @@
...
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:GettingStarted"
+             x:Class="GettingStarted.MainPage" 
+             Padding="0,40,0,0">
+     <StackLayout x:Name="mainLayout">
+       <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+               VerticalOptions="Start" HorizontalOptions="FillAndExpand"
+               Clicked="ClickToShowPopup_Clicked" />
```
Comment (jayaleshwari) [inline MAUI/Popup/getting-started.md:102]: Add SfPopup here as xaml definition.

File: MAUI/Popup/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,287 @@
...
+namespace GettingStarted
+{
+    public partial class MainPage : ContentPage
+    {
+        SfPopup popup;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            popup = new SfPopup();
```
Comment (jayaleshwari) [inline MAUI/Popup/getting-started.md:121]: can avoid this

File: MAUI/Popup/overview.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+
+## Key features
+
```
Comment (jayaleshwari) [inline MAUI/Popup/overview.md:15]: Can take key features from whats new or release notes

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
Please address my review comments

File: MAUI/Popup/getting-started.md (hunk 18 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:102]: modified

File: MAUI/Popup/getting-started.md (hunk 19 above)
Comment (RiyashameedM) [reply MAUI/Popup/getting-started.md:121]: modified

File: MAUI/Popup/overview.md (hunk 20 above)
Comment (RiyashameedM) [reply MAUI/Popup/overview.md:15]: Modified

**Review by PiruthivirajM — APPROVED** [review]
approved

**Pull request merged**